core/src/Streamly/Data/ParserK.hs

```diff
@@ -18,27 +18,36 @@ module Streamly.Data.ParserK
 
     -- * Parsers
     -- ** Conversions
-    , fromFold
-    , fromParser
+    , adaptC
     -- , toParser
 
     -- ** Without Input
     , fromPure
     , fromEffect
     , die
+
+    -- * Deprecated
+    , fromFold
+    , fromParser
     )
 
 where
 
 import Control.Monad.IO.Class (MonadIO)
 import Streamly.Internal.Data.Fold (Fold)
 import Streamly.Internal.Data.Unbox (Unbox)
+import Streamly.Internal.Data.Array (Array)
 import qualified Streamly.Internal.Data.Parser as ParserD
 
 import Streamly.Internal.Data.Parser.ParserK.Type
 
--- | Convert a 'Fold' to a 'ParserK'.
---
+{-# DEPRECATED fromFold "Please use \"adaptC . ParserD.fromFold\" instead." #-}
```

Comment:
```
import qualified Streamly.Data.Parser as Parser

fromFold = adaptC . Parser.fromFold
```

Because we are assuming we define `fromFold` in this module. Just like how we use the `Fold.xyz` or `Stream.xyz`
We can qualify both I guess.